tests/test_simple_sample_cur.py

```diff
@@ -0,0 +1,73 @@
+import unittest
+import numpy as np
+
+from sklearn.datasets import load_boston
+from sklearn import exceptions
+
+from skcosmo.sample_selection import SimpleCUR
+
+
+class TestSimpleCUR(unittest.TestCase):
+    def setUp(self):
+        self.X, self.y = load_boston(return_X_y=True)
+        self.idx = [488, 450, 183, 199, 380, 483, 214, 126, 374, 8]
```

Comment:
Is there any reason to use these indices in particular? Would a random number generator work just as well?

Reply:
oh! These are the "ground truth" of  the dataset -- which features should be chosen by CUR in all cases (because it's a deterministic method).